src/main/js/controls/Gantt/helpers/datapointHelpers.js

```diff
@@ -82,6 +82,49 @@ export const renderSelectionPath = (scale, config, path, dataPoint, index) =>
         )
     );
 
+/**
+ * Renders the circle svg element for event which shows up when clicked on the data point.
+ * It is hidden by default and toggled visible onClick.
+ *
+ * @private
```

Comment:
nit: This is also not a private method

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/219/commits/33b5cca47796ca87966a1c265591cd6cc253cb5a#diff-890519d63508b6cb8fa8894a8026b166R88)